broker/src/main/java/io/camunda/zeebe/broker/Broker.java

```diff
@@ -156,13 +162,21 @@ private StartProcess initStart() {
     final StartProcess startContext = new StartProcess("Broker-" + localBroker.getNodeId());
 
     startContext.addStep("Migrated Startup Steps", this::migratedStartupSteps);
-    startContext.addStep("membership and replication protocol", () -> atomixCreateStep(brokerCfg));
     startContext.addStep(
         "command api transport and handler",
         () -> commandApiTransportAndHandlerStep(brokerCfg, localBroker));
     startContext.addStep("subscription api", () -> subscriptionAPIStep(localBroker));
 
-    startContext.addStep("cluster services", () -> clusterServices.start().join());
```

Comment:
:thought_balloon: I'm surprised changing the order of things had no effect, but I guess after so much refactoring it's possible